src/clarity-angular/layout/tabs/tabs.spec.ts

```diff
@@ -54,7 +55,7 @@ describe("Tabs", () => {
         TestBed.configureTestingModule({
             imports: [ClrTabsModule],
             declarations: [TestComponent],
-            providers: [TabLink, TabContent]
+            providers: [Tabs, IfActiveService]
```

Comment:
Oops, that shouldn't be there. I replaced it with `TabsService` which is the correct provider it needs.